src/commands/init.js

```diff
@@ -169,6 +169,11 @@ const processInitForm = async (
         return value
       },
     },
+    // TODO:
+    //
+    // protocols that don't support contract
+    // - arweave
+    // - tendermint
```

Comment:
I was trying to init an arweave subgraph example using `graph-cli` but the program crashed at this point, so I just left a `TODO` doc on this